### DISCUSSION TITLE
[#995617] Easy way to edit "Player" position

### DISCUSSION THREAD

**Discussion by ihategravel2** [Mojang/minecraft-editor#27] [Feedback]
Right now, if you want to get somewhere in the world, you have to fly there manually or go into Crosshair Mode and teleport yourself via command. I suggest adding the ability to click on the "Pos" coordinates at the bottom left-hand corner and manually change them. You would, of course, be teleported to the new position.

If this sounds familiar, it's because Amulet Editor has basically the same thing, and it is very useful.

**Comment by minecraftchloe** [comment]
I _think_ it might be possible to make this as an extension, but it's also something we have prototyped internally! You can also use /commands in Tool Mode.

**Reply by ihategravel2** [reply]
Did not even realize you could bring up chat in Tool Mode. 👍

**Comment by dacowan** [comment]
Doesn’t the player position extension already allow you to type in new coordinates?

**Reply by ihategravel2** [reply]
I've never seen a extension of such a thing.

**Reply by minecraftchloe** [reply]
We've mocked out having editable coordinates in the bottom status bar, and one of our team members also made an extension that does this! It's a bit of sidequest for us to update the bottom status bar to have editable fields at the moment, but let me look into us adding the player position extension we made a while ago _back_ into the Editor for everyone!

**Comment by minecraftchloe** [comment]
Check out the [GoToMark Extension sample](https://github.com/Mojang/minecraft-editor-extension-samples)! Possible with the release of #130  :)
